packages/gp2-components/src/organisms/WorkingGroupCard.tsx

```diff
@@ -0,0 +1,93 @@
+import { gp2 as gp2Model } from '@asap-hub/model';
+import {
+  Card,
+  ceruleanFernGradientStyles,
+  lead,
+  LinkHeadline,
+  Paragraph,
+  pixels,
+  Subtitle,
+  utils,
+} from '@asap-hub/react-components';
+import { gp2 } from '@asap-hub/routing';
+import { css } from '@emotion/react';
+import usersIcon from '../icons/users-icon';
+import IconWithLabel from '../molecules/IconWithLabel';
+
+const { rem } = pixels;
+const { workingGroups } = gp2;
+const { getCounterString } = utils;
+
+type WorkingGroupCardProps = Pick<
+  gp2Model.WorkingGroupResponse,
+  'id' | 'title' | 'members' | 'shortDescription' | 'leadingMembers'
+>;
+const containerStyles = css({
+  display: 'flex',
+  gap: rem(16),
+  margin: rem(24),
+});
+
+const textContainerStyles = css({
```

Comment:
we are doubling the display flex
I think we can merge both to a single containerStyles

Reply:
done